src/agentscope/studio/static/js/gallery.js

```diff
@@ -248,14 +261,11 @@ function showLoadWorkflowList(tabId) {
       if (!Array.isArray(data.files)) {
         throw new TypeError("The return data is not an array");
       }
-
       const container = document.getElementById(tabId).querySelector(".grid-container");
       container.innerHTML = "";
-
-      data.files.forEach(workflowName => {
-        const title = workflowName.replace(/\.json$/, "");
-        const thumbnail = generateThumbnailFromContent({ title });
-        createGridItem(title, container, thumbnail, "", "", true);
+      data.files.forEach(fileName => {
+        const thumbnail = generateThumbnailFromContent({title: fileName});
+        createGridItem(fileName, container, thumbnail, "", "", true); // 传递完整的文件名
```

Comment:
please delete the comment or translate it to English (same for the comments below)